roxctl/helm/output/output.go

```diff
@@ -108,6 +111,15 @@ func (cfg *helmOutputCommand) outputHelmChart() error {
 		return errors.Wrap(err, "unable to get chart meta values")
 	}
 
+	if version.IsReleaseVersion() || pkgEnv.TelemetryStorageKey.Setting() != "" {
```

Comment:
Nit: I would also add check for `DISABLED` here to eliminate `TelemetryEnabled=true` and `TelemetryStorageKey=DISABLED` combination